x-pack/qa/saml-idp-tests/build.gradle

```diff
@@ -17,11 +20,29 @@ testFixtures.useFixture ":x-pack:test:idp-fixture"
 
 String outputDir = "${project.buildDir}/generated-resources/${project.name}"
 task copyIdpCertificate(type: Copy) {
-    from idpFixtureProject.file('idp/shibboleth-idp/credentials/idp-browser.pem');
+    from idpFixtureProject.files('idp/shibboleth-idp/credentials/idp-browser.pem', 'idp/shibboleth-idp/metadata/idp-metadata.xml');
     into outputDir
 }
 project.sourceSets.test.output.dir(outputDir, builtBy: copyIdpCertificate)
-integTestCluster.dependsOn copyIdpCertificate
+
+task setupPorts {
```

Comment:
Thanks for noticing and fixing this @jaymode !
The code to look up the ports is not necessary  `TestFixturesPlugin` already sets this up,
`postProcessFixture` will have extension properties with the ports, so you just need something like this:  `postProcessFixture.doLast {  println ext."test.fixtures.shibboleth-idp.tcp.443"  }` Note that any testing task also gets passed a system property by the same name.

Reply:
thanks for that tip!